src/components/Stakes.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <q-card v-if="entity && entity.currentStakes.length" class="info-table">
+  <q-card v-if="entity && entity.currentStakes && entity.currentStakes.length" class="info-table">
```

Comment:
Minor, but could be shortened to `v-if="entity?.currentStakes?.length"`.

Reply:
There are a few like this that need to be adjusted. I'll do that in a different PR.